test/unit/api/sort_black.h

```diff
@@ -114,11 +114,16 @@ void SortBlack::testDatatypeSorts()
   // get tester
   Term isConsTerm = dcons.getTesterTerm();
   TS_ASSERT(isConsTerm.getSort().isTester());
+  TS_ASSERT(isConsTerm.getSort().getTesterDomainSort() == dtypeSort);
+  Sort booleanSort = d_solver.getBooleanSort();
+  TS_ASSERT(isConsTerm.getSort().getTesterCodomainSort() == booleanSort);
 
```

Comment:
Could you please add test cases for calling `getTesterDomainSort`/`getTesterCodomainSort` on the wrong sort?
```suggestion
TS_ASSERT_THROWS(booleanSort.getTesterDomainSort(), CVC4ApiException&);
TS_ASSERT_THROWS(booleanSort.getTesterCodomainSort(), CVC4ApiException&);

```